packages/web-react/src/components/Card/__tests__/useCardMediaStyleProps.test.ts

```diff
@@ -0,0 +1,126 @@
+import { cssVariablePrefix } from '@lmc-eu/spirit-design-tokens';
+import { renderHook } from '@testing-library/react';
+import { type SpiritCardMediaProps } from '../../../types';
+import { useCardMediaStyleProps } from '../useCardMediaStyleProps';
+
+describe('useCardMediaStyleProps', () => {
+  it('should return defaults', () => {
+    const props: Partial<SpiritCardMediaProps> = {};
+    const { result } = renderHook(() => useCardMediaStyleProps(props));
+
+    expect(result.current.classProps).toBe('');
+    expect(result.current.styleProps).toEqual({});
+  });
+
+  // handle `|| {}` fallback branch
+  it('should return defaults with props `{}` fallback', () => {
+    const props = undefined;
+    const { result } = renderHook(() => useCardMediaStyleProps(props as unknown as SpiritCardMediaProps));
+
+    expect(result.current.classProps).toBe('');
+    expect(result.current.styleProps).toEqual({});
+  });
+
+  it('should return background color classProps', () => {
+    const props: Partial<SpiritCardMediaProps> = {
+      backgroundColor: 'primary',
+    };
+    const { result } = renderHook(() => useCardMediaStyleProps(props));
+
+    expect(result.current.classProps).toBe('bg-primary');
+    expect(result.current.styleProps).toEqual({});
+  });
+
+  it('should return secondary background color classProps', () => {
+    const props: Partial<SpiritCardMediaProps> = {
+      backgroundColor: 'secondary',
+    };
+    const { result } = renderHook(() => useCardMediaStyleProps(props));
+
+    expect(result.current.classProps).toBe('bg-secondary');
+    expect(result.current.styleProps).toEqual({});
+  });
+
+  it('should return tertiary background color classProps', () => {
+    const props: Partial<SpiritCardMediaProps> = {
+      backgroundColor: 'tertiary',
+    };
+    const { result } = renderHook(() => useCardMediaStyleProps(props));
+
+    expect(result.current.classProps).toBe('bg-tertiary');
+    expect(result.current.styleProps).toEqual({});
+  });
```

Comment:
Is it necessary to check all of these priority color variants? The component supports emotion and accent colors as well, anyway the unit tests doesn't have to cover all of the possible combinations, only the crucial minimum one.
I think that you can minimise the number of test cases.